### ISSUE TITLE
DevToys 2.0.1.0 won't open on macOS ARM64

### ISSUE THREAD

**Issue by krehel** [DevToys-app/DevToys#1183]
### Current behavior

macOS states the application is damaged and won't open.  Tried installing from Homebrew, and download directly from GitHub.

### How to reproduce it (as minimally and precisely as possible)

Try to run the application on macOS.  ~~Unsure if Intel chips are affected, but unable to run on ARM64 arch.~~ Edit: the Intel version can be opened on Apple Silicon without issue.

### Expected behavior

App opens and functions normally.

### Screenshots

<img width="372" alt="devtoys" src="https://github.com/DevToys-app/DevToys/assets/39449589/0ab05300-66b1-4097-b19b-12536de4a38a">

### Affected platforms

macOS

### Affected DevToys kind

DevToys (app with GUI)

### DevToys Version

2.0.1.0 ARM64

**Comment by krehel** [comment]
For anyone else coming across this -

xattr -r -d com.apple.quarantine /Applications/DevToys.app will fix the issue.

**Comment by krehel** [comment]
Hi @veler - are you planning to validly sign the macOS application?  This behavior may cause issues to be opened at Homebrew side.

**Comment by veler** [comment]
Hi @krehel,
Thanks for reaching out. I did reproduce this issue this morning too. Interestingly I didn't reproduce it when testing it before.

Is signing the app the only way to go from here? 🤔

**Comment by dineiar** [comment]
I have just installed it from Homebrew, and it is working flawlessly (Apple M2).

Output from brew install:
```
$ brew install --cask devtoys
==> Downloading https://github.com/DevToys-app/DevToys/releases/download/v2.0.1.0/devtoys_osx_arm64.zip
==> Downloading from https://objects.githubusercontent.com/github-production-release-asset-2e65be/411493648/0dd4fbb6-96a7-48f1-963b-7ec36a39edc9?X-Amz-Algorithm=AWS4-HMAC-SHA256&
########################################################################################################################################################################### 100.0%
==> Installing Cask devtoys
==> Moving App 'DevToys.app' to '/Applications/DevToys.app'
Password:
🍺  devtoys was successfully installed!
```

**Comment by sunshine-syz** [comment]
Got the same error after upgrading to the v2.0.1.0 version. (Apple M1)

**Comment by krehel** [comment]
Hi @veler - yeah ultimately you should sign and notarize the application to keep Apple happy.  You might not have seen it if you're testing locally with your own ad-hoc signing.

**Comment by veler** [comment]
> I have just installed it from Homebrew, and it is working flawlessly (Apple M2).
> 
> Output from brew install:
> 
> ```
> $ brew install --cask devtoys
> ==> Downloading https://github.com/DevToys-app/DevToys/releases/download/v2.0.1.0/devtoys_osx_arm64.zip
> ==> Downloading from https://objects.githubusercontent.com/github-production-release-asset-2e65be/411493648/0dd4fbb6-96a7-48f1-963b-7ec36a39edc9?X-Amz-Algorithm=AWS4-HMAC-SHA256&
> ########################################################################################################################################################################### 100.0%
> ==> Installing Cask devtoys
> ==> Moving App 'DevToys.app' to '/Applications/DevToys.app'
> Password:
> 🍺  devtoys was successfully installed!
> ```

Hi @dineiar ,
Thanks for sharing. Have you not get the same error message than other folks here? Do you have the quarantine disabled or something? 🤔

**Comment by veler** [comment]
> Hi @veler - yeah ultimately you should sign and notarize the application to keep Apple happy. You might not have seen it if you're testing locally with your own ad-hoc signing.

That may be it! Well, yikes. I'm brand new on Mac development and I'm learning the hard way I guess 😅
As a temporary workaround until I get it sorted, the only thing I see is to recommend to get the app out of quarantine for now using `xattr -r -d com.apple.quarantine /Applications/DevToys.app`. But definitely not a good long term solution.

**Comment by dineiar** [comment]
> Hi @dineiar , Thanks for sharing. Have you not get the same error message than other folks here? Do you have the quarantine disabled or something? 🤔

I am not getting any error message. I did not remember disabling it, but yes, it seems I have quarantine disabled:
```
$ xattr -p com.apple.quarantine /Applications/DevToys.app
xattr: /Applications/DevToys.app: No such xattr: com.apple.quarantine
```

Sorry for the confusion.

**Comment by matteocontrini** [comment]
According to Apple the message means that:

>If macOS detects that software has been modified or damaged, your Mac notifies you that the app can't be opened. The app might be broken or corrupted, or it might have been tampered with.

I don't see how this has anything to do with signing? That should show another kind of alert.

https://support.apple.com/en-us/102445

**Comment by krehel** [comment]
Fair point.  I took some time to dig into this a bit.  I am not an expert _at all_, just standing on the shoulders of giants and interpreting their writing.  Someone who is an expert can come along and tell me how wrong I am. 😄 

tl;dr the `arm64` is signed ad-hoc by the linker, but the `x64` has no signature at all.  This appears to result in the `arm64` version having a broken signature.

Can see this on the command line -

```
> codesign -v -vvvv --strict DevToys\ ARM64.app
DevToys ARM64.app: code has no resources but signature indicates they must be present
```

```
> codesign -v -vvvv --strict DevToys\ X64.app
DevToys X64.app: code object is not signed at all
In architecture: x86_64
```

My comment is that signing and notarizing the application (correctly), making Gatekeeper happy, Apple happy, etc. would prevent this issue, but it's still possible to damage the app.  Or don't sign it at all, but that just makes no one happy. 😄 

---

Extra information below provided using Apparency.  It is a fantastic tool. Screenshots below from the tool evaluating the `2.0.1` version of DevToys ⬇️ 

---
ARM64

<img width="1212" alt="devtoys arm64" src="https://github.com/DevToys-app/DevToys/assets/39449589/d2e9f8b5-a218-4061-addb-a72b541a4c61">

---
X64

<img width="1212" alt="devtoys x64" src="https://github.com/DevToys-app/DevToys/assets/39449589/82c1272c-9713-4fed-affc-50ee6142163d">

---

All things are the same, minus the "Signed By" (which we covered at the top showing that the ARM64 version has attempted to be signed, and the X64 version isn't sign at all).  No notarization, no gatekeeper, etc.

[Definition](https://www.mothersruin.com/software/Apparency/faq.html#group-components) taken from Mothers Ruin who made Apparency.

_Signed By - The component was automatically signed by the macOS linker. This behavior was introduced in macOS 11 (Big Sur), to [fulfill the Apple Silicon requirement](https://developer.apple.com/documentation/macos-release-notes/macos-big-sur-11_0_1-universal-apps-release-notes/#Code-Signing) that all executables have some sort of code signature — and thus have a code signing digest (or cdhash). This linker signature is a form of ad-hoc signature (and so doesn't tell you anything about the developer's identity); it is usually applied only to the Apple Silicon architecture (with the Intel architecture being left unsigned). This automatic signing is useful only for standalone Mach-O executables, since it doesn't seal any of the bundle resources that would be covered by a codesign(1)-generated signature (even an ad-hoc one)._

So, good for standalone binaries, not so much for app bundles.

Which as I would interpret this, the signature failure joined with the quarantine flag is what makes it broken.  Because when quarantine is stripped, the application works fine.

_Broken_ signature + quarantine: App is damaged, don't use it
No signature + quarantine: be careful at your own risk
_Broken_ or no signature + quarantine removed: be careful at your own risk

**Comment by Sly777** [comment]
> > Hi @dineiar , Thanks for sharing. Have you not get the same error message than other folks here? Do you have the quarantine disabled or something? 🤔
> 
> I am not getting any error message. I did not remember disabling it, but yes, it seems I have quarantine disabled:
> 
> ```
> $ xattr -p com.apple.quarantine /Applications/DevToys.app
> xattr: /Applications/DevToys.app: No such xattr: com.apple.quarantine
> ```
> 
> Sorry for the confusion.

I also tried the homebrew way to install devtoys but I got quarantine error. (M3 Pro, Sonoma 14.5) I also tried to install manually but It also got the same error.

**Comment by tobgen** [comment]
Same here, MacOS 14.5, M2 Pro

**Comment by nathanlesage** [comment]
Hi, chiming in as I wanted to try it out but couldn't due to apparent reasons. Making Apple happy is fortunately comparatively easy BUT comes at a cost. You'll first need a 99$/year Apple Developer subscription, which will give you a code signing certificate. That you need to do first using codesign. Second, you then also need to send the codesigned binary to Apple's notarization server. Once that thing is back, you can distribute that app bundle.

It's a bit convoluted and ill-documented, but Apple indeed makes it easier than Microsoft in complying with the platform requirements.

Incidentally, googling a bit for "Electron notarizing" gives quite good results, but I don't know how it works with a non-Electron app (but I guess the tooling can be adapted)

**Comment by veler** [comment]
Hi there,
I just wanted to give a quick update: I bought the Apple Developer subscription yesterday. I'm waiting for the payment to go through. After that I will try to sign the code as soon as possible. This issue is currently my top 1 priority.
Please be patient and sorry for the inconvenience :)

**Comment by krehel** [comment]
No worries @veler - thank you!  Let me know if there's anything else I can do to help you out.

**Comment by iparr** [comment]
@veler - do you have a 'Ko-fi' or something similar so that those of us who appreciate it can give you some token amount to offset the cost of your Apple Developer subscription?

**Comment by randelramirez** [comment]
> Hi there, I just wanted to give a quick update: I bought the Apple Developer subscription yesterday. I'm waiting for the payment to go through. After that I will try to sign the code as soon as possible. This issue is currently my top 1 priority. Please be patient and sorry for the inconvenience :)

No worries, thank you for making it cross-platform 🙏

**Comment by veler** [comment]
More than 48h later, Apple Developer account activation is still "pending". Dang it !

**Comment by nathanlesage** [comment]
> More than 48h later, Apple Developer account activation is still "pending". Dang it !

Don't fret, once it's through you're gonna have the cert for ever without issues (in my experience at least)!

Ah, and one additional tip that is less obvious: from time to time, Apple will update its TOS and won't send you an email about that. The point at which you realize this is when notarization will fail randomly, and the error reads something along the lines of"you didn't agree to XYZ", that'll be a bit annoying but other than that it should be smooth once set up properly.

**Comment by veler** [comment]
> > More than 48h later, Apple Developer account activation is still "pending". Dang it !
> 
> Don't fret, once it's through you're gonna have the cert for ever without issues (in my experience at least)!
> 
> Ah, and one additional tip that is less obvious: from time to time, Apple will update its TOS and won't send you an email about that. The point at which you realize this is when notarization will fail randomly, and the error reads something along the lines of"you didn't agree to XYZ"*, that'll be a bit annoying but other than that it should be smooth once set up properly.
> 
> * That is when you'll have to actually log in and click accept on their TOS, which can be annoying

Thanks for the tip! 😁

**Comment by veler** [comment]
Update: I just called Apple support and apparently the account is still "pending" because they do a manual review when you subscribe, and they are a little bit behind because they got many more people with WWDC24. Just need to be extra patient :-)

**Comment by veler** [comment]
Alright, my Apple Developer account got activated earlier today. I followed some tutorial for .NET. Looks like the code is being signed correctly, unless contradicted by Notarization, which is running for about an hour now. We're getting there :D

**Comment by veler** [comment]
Alright. Notarization gone for about 16h and finished on an error. Looking at `xcrun notarytool log`, I found the following:

```json
  "issues": [
    {
      "severity": "error",
      "code": null,
      "path": "devtoys_osx_arm64.zip/DevToys.app/Contents/MacOS/DevToys",
      "message": "The signature of the binary is invalid.",
      "docUrl": "https://developer.apple.com/documentation/security/notarizing_macos_software_before_distribution/resolving_common_notarization_issues#3087735",
      "architecture": "arm64"
    }
  ]
```

Looking at `codesign -vvv --deep --strict` as suggested by the documentation, I got the following:

```
DevToys.app/Contents/MacOS/DevToys: a sealed resource is missing or invalid
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/[Content_Types].xml
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/LICENSE.md
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/DevToys.Tools.nuspec
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/tr-TR/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/sl/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/zh-Hans/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/zh-Hans/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/zh-Hant/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/zh-Hant/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/fr-FR/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/Microsoft.IdentityModel.Logging.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/de-DE/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/pl-PL/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/pl/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/vi/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/vi-VN/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/da-DK/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/sv/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/el-GR/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/es-ES/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/ColorCode.Core.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/da/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/ru-RU/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/ca-ES/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/ja/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/es-AR/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/it/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/es-MX/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/ar-SA/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/ru/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/en-GB/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/zxing.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/kn-IN/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/Markdig.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/hu-HU/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/nl-NL/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/ColorCode.HTML.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/ro/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/Cronos.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/ja-JP/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/ro-RO/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/hi-IN/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/pt/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/he-IL/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/he-IL/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/YamlDotNet.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/uk/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/Microsoft.IdentityModel.JsonWebTokens.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/cs-CZ/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/Microsoft.IdentityModel.Abstractions.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/pt-PT/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/Markdown.ColorCode.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/DevToys.Tools.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/Newtonsoft.Json.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/ko-KR/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/pt-BR/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/hu/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/nl/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/it-IT/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/kw-GB/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/nb/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/nb/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/de/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/id-ID/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/ko/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/fi/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/ta-IN/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/fr/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/sv-SE/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/Microsoft.IdentityModel.Tokens.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/es/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/Microsoft.IO.RecyclableMemoryStream.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/fa/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/te-IN/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/bn-BD/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/uk-UA/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/DevToys.Tools.deps.json
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/CronExpressionDescriptor.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/ZXing.ImageSharp.V3.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/fi-FI/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/tr/CronExpressionDescriptor.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/af-ZA/DevToys.Tools.resources.dll
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/DevToys.Tools.pdb
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/lib/net8.0/DevToys.Tools.runtimeconfig.json
file added: DevToys.app/Contents/Resources/Plugins/DevToys.Tools/package/services/metadata/core-properties/dbe3ac33bf7642a0a54a00958171a476.psmdcp
```

Two things about it:
1. `a sealed resource is missing or invalid` doesn't run a bell to me. Is the issue with one of the files in `Resources` folder?
2. Files in `DevToys.app/Contents/Resources/Plugins/DevToys.Tools/` folder are not signed. They are added post-build to the `.app`. Am I right to assume it's the root cause of the problem?

**Comment by veler** [comment]
Looks like my assumption was correct. I succeeded to notarize the app! I'm gonna push an update tonight or tomorrow. Thanks for your patience!

**Comment by veler** [comment]
Update is available here: https://github.com/DevToys-app/DevToys/releases/tag/v2.0.2.0
And soon on Homebrew.
Thanks everyone! Please let me know if the issue keeps reproducing for you.

**Comment by chawax** [comment]
It works perfect now, thanks a lot !

**Comment by vineeth-au** [comment]
Works perfetly... thank you for taking the time and fixing the issue.

**Comment by rjorgenson** [comment]
v2.0.2.0 made it to homebrew and works perfectly now, thanks!
